docs/sources/dashboards/manage-dashboards/index.md

```diff
@@ -159,15 +172,15 @@ By default, Grafana queries your data source every 30 seconds. Setting a low ref
 
 We recommend the following:
 
-- Do not enable auto-refreshing on dashboards, panels, or variables unless you need it. Users can refresh their browser manually, or you can set the refresh rate for a time period that makes sense (every ten minutes, every hour, and so on).
-- If it is required, then set the refresh rate to once a minute. Users can always refresh the dashboard manually.
+- Don't enable auto-refreshing on dashboards, panels, or variables unless you need it. Users can refresh their browser manually, or you can set the refresh rate for a time period that makes sense (every ten minutes, every hour, and so on).
```

Comment:
It feels strange to read a recommendation not to do something. I wonder if this can be worded in the positive?

Reply:
I'll circle back to this. I was was just fixing the Vale error here, but I need more information from the squad to make a good substantive edit here.

Reply:
right, got it. No need to handle in this PR, just something that jumped out to me. All good!

Reply:
```suggestion
- Only enable auto-refreshing on dashboards, panels, or variables unless if necessary. Users can refresh their browser manually, or you can set the refresh rate for a time period that makes sense (every ten minutes, every hour, and so on).
```